setup.py

```diff
@@ -169,7 +169,7 @@ def run(self):
         if get_os_name() != 'win':
             num_threads = os.getenv('BUILD_NUM_THREADS',
                                     multiprocessing.cpu_count())
-            build_args += ['--', f'-j{num_threads}']
+            build_args += ['--', f'-j24']
```

Comment:
Accidentally included this?

Reply:
My PC often shuts down due to overheat when I use all the threads... will revert